### ISSUE TITLE
implement cross fold validation

### ISSUE THREAD

**Issue by clarkegavin** [clarkegavin/roblox_text_analysis#12]
implement cross fold validation and test evaluation

cross fold validation on the training set
standard evaluation on the test set
